modules/programs/jujutsu.nix

```
@@ -45,7 +53,26 @@ in {
     home.packages = [ cfg.package ];
 
     home.file.".jjconfig.toml" = mkIf (cfg.settings != { }) {
-      source = tomlFormat.generate "jujutsu-config" cfg.settings;
+      source = tomlFormat.generate "jujutsu-config" (cfg.settings
+        // optionalAttrs (cfg.ediff) (let
+          emacs_diff_script = pkgs.writeScriptBin "emacs-ediff" ''
+            #!${pkgs.stdenv.shell}
+            set -euxo pipefail
+            ${config.programs.emacs.package}/bin/emacsclient -c --eval "(ediff-merge-files-with-ancestor \"$1\" \"$2\" \"$3\" nil \"$4\")"
+          '';
+        in {
+          merge-tools.ediff = {
+            program = "sh";
```

Perhaps
```suggestion
            program = getExe emacsDiffScript;
```
would work? I assume it would simplify merge-args to

``` nix
merge-args = [ "$left" "$right" "$base" "$output" ];
```